CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 ### Migrations and checks
 #### Schema migrations
 #### Data migrations
+- All bounding boxes are cropped to not exceed (-20026376.39, -20048966.10, 20026376.39, 20048966.10) (in EPSG:3857)
```

Comment:
All bounding boxes are cropped not to exceed world boundaries ([-20026376.39, -20048966.10, 20026376.39, 20048966.10] in EPSG:3857).